editor.planx.uk/src/pages/FlowEditor/components/Settings/ServiceSettings.tsx

```diff
@@ -200,25 +221,82 @@ const ServiceSettings: React.FC = () => {
             type="submit"
             variant="contained"
             color="primary"
-            disabled={!formik.dirty}
+            disabled={!elementsForm.dirty}
           >
             Update elements
           </Button>
         </EditorRow>
-        <Snackbar
-          open={isAlertOpen}
-          autoHideDuration={6000}
-          onClose={handleClose}
-        >
-          <Alert
-            onClose={handleClose}
-            severity="success"
-            sx={{ width: "100%" }}
-          >
-            Service settings updated successfully
-          </Alert>
-        </Snackbar>
-      </form>
+      </Box>
+      <Box component="form" onSubmit={statusForm.handleSubmit}>
+        <EditorRow>
+          <Typography variant="h2" component="h3" gutterBottom>
+            Status
+          </Typography>
+          <Typography variant="body1">
+            Manage the status of your service.
+          </Typography>
+        </EditorRow>
+        <EditorRow background>
+          <FormControlLabel
+            label={statusForm.values.status}
+            sx={{
+              [`& .${formControlLabelClasses.label}`]: {
+                fontWeight: FONT_WEIGHT_BOLD,
+                textTransform: "capitalize",
+                fontSize: 19,
+              },
+            }}
+            control={
+              <Switch
+                name="service.status"
+                color="primary"
+                checked={statusForm.values.status == "online"}
+                onChange={() =>
+                  statusForm.setFieldValue(
+                    "status",
+                    statusForm.values.status == "online" ? "offline" : "online",
+                  )
+                }
+              />
+            }
+          />
+          <Typography variant="body1">
+            Toggle your service between "offline" and "online".
+          </Typography>
+          <Typography variant="body1">
+            A service must be online to be accessed by the public, and to enable
+            analytics gathering.
```

Comment:
nit: I wonder if it's worth adding one more sentence here like "Offline services can still be edited and published as normal." to encourage good maintenance?